package.json

```diff
@@ -35,11 +37,12 @@
   },
   "scripts": {
     "docker:build": "yarn workspace @nordeck/matrix-neoboard-widget docker:build",
-    "start": "yarn workspace @nordeck/matrix-neoboard-widget start",
     "dev": "yarn workspace @nordeck/matrix-neoboard-widget dev",
+    "dev:https": "yarn workspace @nordeck/matrix-neoboard-widget dev:https",
     "build": "yarn workspaces run build",
-    "test": "yarn workspaces run test",
-    "test:all": "yarn workspaces run test:all",
+    "test": "vitest watch",
+    "test:all": "vitest run --coverage",
+    "test:update_snapshots": "vitest -u",
```

Comment:
Not sure if we should put this into the scripts list. There are some basic commands to run tests. If you need advanced stuff you should know what you are doing. Otherwise the list gets longer and longer.